.gitignore

```diff
@@ -20,3 +20,7 @@ relic/
 .pytest_cache
 results.xml
 pip-wheel-metadata/
+.DS_Store
+.project
+.pydevproject
+.vscode
```

Comment:
You'll want to add `*code-workspace` to `.gitignore` and a blank line at the end.

And remove the `*.code-workspace` files included in this PR.  We don't want these committed to the repo.

Reply:
Done